### PR TITLE
readme updates

### PR DESCRIPTION
**Pull request by TheMP**
Small readme updates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,13 +48,15 @@ To create a conda environment for Python 3.7
...
+If the conda setup is causing too many problems, we also provide a Docker image with everything set up for you and ready-to-use python notebooks with demo examples.
+To run the example notebooks without installing our libraries natively on your machine, you can use our Docker image:
 ```
 cd scripts
 ./build_docker.sh && ./run_docker.sh
 ```
 Then copy and paste the URL provided by the docker container into your browser to access Jupyter notebook.
-This requires a Docker install.
+
+For this setup to work you need to have a Dockr service installed. You can get it [here](https://docs.docker.com/get-docker/).
```
Comment (radujica) [inline README.md:59]: Docker typo?

File: README.md [hunk 2]
```diff
@@ -43,13 +48,15 @@ To create a conda environment for Python 3.7
...
+If the conda setup is causing too many problems, we also provide a Docker image with everything set up for you and ready-to-use python notebooks with demo examples.
+To run the example notebooks without installing our libraries natively on your machine, you can use our Docker image:
 ```
 cd scripts
 ./build_docker.sh && ./run_docker.sh
 ```
 Then copy and paste the URL provided by the docker container into your browser to access Jupyter notebook.
-This requires a Docker install.
+
+For this setup to work you need to have a Docker service installed. You can get it [here](https://docs.docker.com/get-docker/).
```
Comment (endrjuskr) [inline README.md:59]: is it docker `client`?

File: README.md [hunk 3]
```diff
@@ -43,13 +48,15 @@ To create a conda environment for Python 3.7
 
 #### Running the examples only, without installing:
 
-To run the example notebooks without installing, using Docker, you can also run: 
+If the conda setup is causing too many problems, we also provide a Docker image with everything set up for you and ready-to-use python notebooks with demo examples.
+To run the example notebooks without installing our libraries natively on your machine, you can use our Docker image:
 ```
 cd scripts
 ./build_docker.sh && ./run_docker.sh
```
Comment (endrjuskr) [inline README.md:55]: Probably here we can use `run_docker.sh`? User doesn't need to build image.

File: README.md (hunk 2 above)
Comment (TheMP) [reply README.md:59]: it comes together, both client and server

**Pull request merged**